gui/control.js

```diff
@@ -71,6 +95,16 @@ function render() {
             showToast("Error!", errorToMessage[error])
         });
     }
+
+    const userPictures = document.querySelectorAll(".user-picture");
+    userPictures.forEach(function(picture) {
+        const username = picture.dataset.username;
+        picture.textContent = getInitials(username);
+    });
```

Comment:
instead of looping and changing things, why not do it at the time account cards are created